### PR TITLE
Feature/data cleanup

### PR DESCRIPTION
**Pull request by tatianaakim**
Add python code that cleans up raw BANES car parking data, then integrates it with Events, Rugby and weather data

### CODE REVIEW

File: python/cleanup/main.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+import pandas as pd
+import numpy as np
+import utilities as ut
+import transform_records as tr
+import time as tm
+
+
+# import data 
+rawData = pd.read_csv('C:/Users/Tatiana Kim/Dropbox (Iridium Insights)/Local Folder Tatiana/ML/ParkingProject/BANES_Historic_Car_Park_Occupancy.csv')
```
Comment (kinbiko) [inline python/cleanup/main.py]: Can we make this a relative path please. E.g.
`../../data/BANES_Historic_Car_Park_Occupancy.csv`

It easier to create this folder than setting up windows with a user called Tatiana and a dropbox folder for Iridium Insights :)

File: python/cleanup/main.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+# Just want to extract Date and Time from LastUpdate - 
+# we will do some joins on Date, so need this as a separate column
+OccupancyDF['Date'] = OccupancyDF['LastUpdate'].dt.date
+OccupancyDF['Time'] = OccupancyDF['LastUpdate'].dt.time
+
+
+##########################################################################
+#  Read external data:
+
+RugbyDF = pd.read_csv('Rugby_events.csv')
```
Comment (kinbiko) [inline python/cleanup/main.py]: Can we pretend that these csv files exist in a folder called `../../data/`. see above.

File: python/cleanup/transform_records.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    return refinedDF
+
+
+#' Remove duplicate records 
+#' 
+#' @param x Data frame containing records.
+#' @param first_upload If TRUE, ensures record with oldest DateUploaded is kept.
+
+
+#def deduplicate(x, first_upload) :
```
Comment (kinbiko) [inline python/cleanup/transform_records.py]: If this is commented anyway, can we just delete it?

**Review by kinbiko — CHANGES_REQUESTED** [review]

File: python/cleanup/main.py (hunk 1 above)
Comment (tatianaakim) [reply python/cleanup/main.py]: Yes, that makes sense :) Done in the latest commit 8db4cd1a905c7ef7b965233e77a3f4edd108c8bb

**Comment by tatianaakim** [comment]
I've now also added updates to python README file as well as cleanup README file - the intro pages look a bit bare otherwise

**Pull request merged**